uisdk/src/main/res/layout/uisdk_booking_checkout_view.xml

```diff
@@ -64,16 +64,39 @@
                     android:layout_weight="1" />
             </LinearLayout>
 
-            <com.karhoo.uisdk.screen.booking.checkout.comment.BookingOptionalInfoView
-                android:id="@+id/bookingRequestFlightDetailsWidget"
+            <LinearLayout
+                android:id="@+id/bookingCheckoutTravelDetailsLayout"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginStart="@dimen/kh_uisdk_spacing_small"
                 android:layout_marginTop="@dimen/kh_uisdk_spacing_small"
-                android:layout_marginEnd="@dimen/kh_uisdk_spacing_small" />
+                android:layout_marginEnd="@dimen/kh_uisdk_spacing_small"
+                android:orientation="horizontal">
+
+                <com.karhoo.uisdk.screen.booking.checkout.component.views.ActionCellView
+                    android:id="@+id/bookingCheckoutTravelDetailsView"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1" />
+            </LinearLayout>
+
+            <LinearLayout
```

Comment:
Why do we need a linear layout for just one nested control? Can we add the margin to the child and remove this parent linear layout?